src/components/Dashboard/TodoList.vue

```diff
@@ -601,6 +605,7 @@ export default {
           status: filing.header.status,
           enabled: Boolean(task.enabled),
           order: task.order,
+          paymentToken: filing.header.paymentToken || null,
```

Comment:
This won't do anything today, but in the future correction payments will need to be resumed/retried.